contracts/src/bridge/Outbox.sol

```diff
@@ -29,8 +29,21 @@ contract Outbox is DelegateCallAware, IOutbox {
     L2ToL1Context internal context;
     uint128 public constant OUTBOX_VERSION = 2;
 
+    modifier outboxIsRunning() {
+        if(context.l2Block == type(uint128).max) revert OutboxNotStart();
+        _;
+    }
```

Comment:
This would be cool if we were deploying a fresh system, but this would be a breaking change.
Contracts querying the outbox could not be expecting this to revert